libs/csm/rest/csm_rest_capacity.py

```diff
@@ -116,38 +187,14 @@ def validate_metrics(self, data, endpoint_param='bytecount'):
                     return False
         return True
 
-    @RestTestLib.authenticate_and_login
-    def get_degraded_capacity(self, endpoint_param='bytecount'):
-        """
-        Get degraded capacity from CSM
-        :param endpoint_param: which endpoint to check for parameters
-        :return : Rest output response
-        """
-        self.log.info("Reading System Capacity...")
-        endpoint = self.config["degraded_cap_endpoint"]
-        if endpoint_param is not None:
-            endpoint = self.config["degraded_cap_endpoint"] + "/" + endpoint_param
-        self.log.info("Endpoint for reading capacity is %s", endpoint)
-        # Fetching api response
-        response = self.restapi.rest_call(request_type="get", endpoint=endpoint,
-                                          headers=self.headers)
-        return response
-
-    def get_degraded_capacity_custom_login(self, header, endpoint_param='bytecount'):
+    def get_degraded_capacity_custom_login(self, header):
```

Comment:
Similar to comment at line 69, we can consider renaming this, so that it is not confused as a call specific to degraded capacity.

Reply:
This is redundant function. It will be removed when I do the rework for F-71E. Kept it for now so that imports dont break.